core-bundle/src/Messenger/WebWorker.php

```diff
@@ -117,18 +117,25 @@ private function processTransport(string $transportName): void
             return;
         }
 
+        $timeLimit = 1;
+
+        if (\function_exists('fastcgi_finish_request') || \function_exists('litespeed_finish_request')) {
+            // Subtract 10 seconds to make sure we never exceed the max execution time
+            $timeLimit = round(min(30, max(1, $this->getRemainingExecutionTime() - 10)));
```

Comment:
Maybe we need to adjust this comment because honestly, we can never be sure. If a message takes longer than 10s to be executed, the process will fail anyway. But if you have such messages, you should probably ensure to only have it being worked on when `$scope` is `cli` and force users to setup a real cronjob/workers.

Reply:
Like so?
```php
// Subtract 10 seconds to reduce the risk of exceeding the max execution time
```